src/exporter-factory.ts

```diff
@@ -13,33 +13,42 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-import * as util from 'util';
+import { inspect } from 'util';
 
 import { SpanExporter } from '@opentelemetry/sdk-trace-base';
-import { Options } from './options';
+import { ExporterOptions, Options } from './options';
 import { Metadata } from '@grpc/grpc-js';
 
 import { OTLPTraceExporter as GRPCTraceExporter } from '@opentelemetry/exporter-trace-otlp-grpc';
 import { OTLPTraceExporter as HTTPTraceExporter } from '@opentelemetry/exporter-trace-otlp-http';
 import { diag } from '@opentelemetry/api';
 
-type SpanExporterFactory = (options: Options) => SpanExporter;
+type SpanExporterFactory = (
+  options: Options,
+  exporterOptions: ExporterOptions
```

Comment:
I think better would be the that exporterOptions will be part of the the Options